src/util/get-property-descriptor.js

```diff
@@ -0,0 +1,35 @@
+const div = document.createElement('div');
+const hasLookupFunctions = !!div.__lookupGetter__;
+
+export default function (obj, key) {
+  if (hasLookupFunctions) {
+    if (obj.constructor.toString().match('Element')) {
```

Comment:
Open to suggestions to make this check more robust. Maybe it's okay for performance to use an `instanceof` check here.